test/tst_masked5.py

```diff
@@ -41,13 +44,20 @@ def test_scaled(self):
 
         f = Dataset(self.testfile)
         v = f.variables["v"]
+        v2 = f.variables["v2"]
         self.assertTrue(isinstance(v[:], ma.core.MaskedArray))
         assert_array_equal(v[:], self.v_ma)
         assert_array_equal(v[2],self.v[2]) # issue #624.
         v.set_auto_mask(False)
         self.assertTrue(isinstance(v[:], np.ndarray))
         assert_array_equal(v[:], self.v)
 
+        # issue 730
+        assert (v2[0]=='first')
```

Comment:
This should probably be `u'first'` for it to pass on both python 2/3. The rule of thumb is to always use bytes/unicode literals (`b'first'` and `u'first'`) unless you want native strings.